### ISSUE TITLE
Question about the equation 3 and 4 in the paper

### ISSUE THREAD

**Issue by Hugo-cell111** [Gank0078/ACR#10]
Hi! I have some questions about the equation 3 and 4:
(1) To equation 3: why the logit of balanced branch towards labeled data should minus the estimate of class prior? In the supplementary meterial, both of two branches use standard CE loss to labeled data.
(2) To equation 4: why the pseudo labels from the standard branch should add the estimate of class prior? I guess the latter is for generating more accurate pseudo labels, am I right?
Looking forward to your reply. Thanks!

**Comment by Gank0078** [comment]
Thank you for your interest in our work! The two questions are related to logit adjustment [1], an effective method for long-tailed learning. Equation 3 and 4 are two forms of logit adjustment. I think after reading [1] can help you understand more clearly about our ACR. In fact, we use standard CE for standard branch and balanced CE for balanced branch for labeled data, which can be seen in https://github.com/Gank0078/ACR/blob/9a8f421ef7508c1e1e13cbae4ca36d442d89e322/train.py#L512 and https://github.com/Gank0078/ACR/blob/9a8f421ef7508c1e1e13cbae4ca36d442d89e322/train.py#L520

[1] Menon A K, Jayasumana S, Rawat A S, et al. Long-tail learning via logit adjustment.
